Master branch is unstable

i got the following error:

```bash
oskar.stark:/Volumes/development/workspaces/SonataCoreBundle (master)$ make test
phpunit
PHP Fatal error:  Uncaught Declaration of Sonata\CoreBundle\Tests\Twig\TokenParser\TemplateBoxNodeTest::testCompile() should be compatible with Twig_Test_NodeTestCase::testCompile($node, $source, $environment = NULL, $isPattern = false)

/Volumes/development/workspaces/SonataCoreBundle/vendor/symfony/phpunit-bridge/DeprecationErrorHandler.php:40
/Volumes/development/workspaces/SonataCoreBundle/Tests/Twig/Node/TemplateBoxNodeTest.php:20

  thrown in /Volumes/development/workspaces/SonataCoreBundle/Tests/Twig/Node/TemplateBoxNodeTest.php on line 20

Fatal error: Uncaught Declaration of Sonata\CoreBundle\Tests\Twig\TokenParser\TemplateBoxNodeTest::testCompile() should be compatible with Twig_Test_NodeTestCase::testCompile($node, $source, $environment = NULL, $isPattern = false)

/Volumes/development/workspaces/SonataCoreBundle/vendor/symfony/phpunit-bridge/DeprecationErrorHandler.php:40
/Volumes/development/workspaces/SonataCoreBundle/Tests/Twig/Node/TemplateBoxNodeTest.php:20

  thrown in /Volumes/development/workspaces/SonataCoreBundle/Tests/Twig/Node/TemplateBoxNodeTest.php on line 20

make: *** [test] Error 255
```

--- a/Tests/Twig/Node/TemplateBoxNodeTest.php
+++ b/Tests/Twig/Node/TemplateBoxNodeTest.php
@@ -38,9 +38,9 @@ class TemplateBoxNodeTest extends \Twig_Test_NodeTestCase
      * @covers Twig_Node_Block::compile
      * @dataProvider getTests
      */
-    public function testCompile($node, $source, $environment = null)
+    public function testCompile($node, $source, $environment = null, $isPattern = false)
     {
-        parent::testCompile($node, $source, $environment);
+        parent::testCompile($node, $source, $environment, $isPattern);
     }
 
     /**

